### PR TITLE
Turn on CICE albedo use by ATM for coupled model; update Prototype test suite to initial P8 configuration; 

### LINKED ISSUE #464: snod bug
## Description
Provide a clear and concise description of what the bug is.
Also give a description of how to fix the bug.
The snow depth in the atmosphere surface file has many bulls eye near the land/ocean boundaries in coupled runs.
It turns out that there is a bug in the way snow volume from the ice model as used in the atmos model. 
### To Reproduce:
What compilers/machines are you seeing this with?
Give explicit steps to reproduce the behavior.
1. do this
2. then that
3. then, oops, look at the bug

## Additional context
Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. Example:
- needs to be fixed also in noaa-emc/nems/issues/<issue_number>
- needed for noaa-emc/fv3atm/pull/<pr_number>

## Output

**Screenshots**
If applicable, drag and drop screenshots to help explain your problem.

**output logs**
If applicable, include relevant output logs.
Either drag and drop the entire log file here (if a long log) or

```
paste the code here (if a short section of log)
```

## Testing:

1. Have you tested the code changes? On what platforms?

2. Have you run regression test in ufs-weather-model or ufs-s2s-model with code changes?
- Will the baseline results change? 
- If the baseline results change, is it expected? Please give brief explanation.

## Dependent PRs:

Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. Example:
- required to support noaa-emc/GFDL_atmos_cubed_sphere/issues/<issue_number>
- ncar/ccpp-physics/pull/<pr_number>
- associated ufs-weather-model/pull/<pr_number>

### LINKED ISSUE #854: Turn on CICE albedos in coupled model and create initial P8 feature set tests
## Description

CICE albedos are available for use in the coupled model after PRs [#723](https://github.com/ufs-community/ufs-weather-model/pull/723), [#701](https://github.com/ufs-community/ufs-weather-model/pull/701). Use of CICE albedos is controlled with namelist parameter ``use_cice_alb``. 

## Solution

Since this feature is planned for inclusion in P8 and the code is already committed, the default_vars ``export_cpl`` should be modified 

```
diff --git a/tests/default_vars.sh b/tests/default_vars.sh
index 14315a13..aaa06d83 100755
--- a/tests/default_vars.sh
+++ b/tests/default_vars.sh
@@ -1120,8 +1120,9 @@ export USE_CICE_ALB=.false.
 export KTHERM=2
 export TFREEZE_OPTION=mushy

+# P8 CICE Albedos
+export USE_CICE_ALB=.true.
```

In ``cpld_bmark_p7`` (and the 35d version), turn the feature off

```
diff --git a/tests/tests/cpld_bmark_p7 b/tests/tests/cpld_bmark_p7
index 62fcdf32..2fd7dda2 100644
--- a/tests/tests/cpld_bmark_p7
+++ b/tests/tests/cpld_bmark_p7
@@ -142,5 +142,6 @@ export RST_END=$RUN_END
 export RST_2_END=$RUN_END

 export USE_CICE_ALB=.false.
 export CCPP_SUITE=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
 export FV3_RUN=cpld_control_run.IN
```

The oRTs should be run using a feature branch containing these changes. If all oRTs pass, a PR should be opened to commit these changes and enable this feature. This will require renaming the tests and updating ``rt.conf``

## Related to

- required to support [P8](https://github.com/ufs-community/ufs-weather-model/projects/7)

### LINKED ISSUE #940: Update Prototype Test suites P7->P8
## Description
The existing P7 test suite for coupled and uncoupled needs to be updated to the initial P8 test suite by enabling RRTMGP.

## Solution

Make the required changes in default_vars and the standalone P7 tests, rename the tests and make the required changes in rt.conf

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [x] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

- Uses two SDFS as initial P8 SDFs. For both standalone and coupled, these are identical to the existing nsstNoahmpUGWPv1 SDFs.
- Changes the default setting of ``USE_CICE_ALB`` from .false. to .true. in ``default_vars``. 
- Changes the names of all ``p7`` tests to ``p8``.
- Retains the ``cpld_bmark_p7`` and ``cpld_control_p7_rrtmgp`` configuration with ``USE_CICE_ALB`` set .false.
- Adds 35d tests for the P8 configuration.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #854 
- fixes https://github.com/NOAA-EMC/fv3atm/issues/464

## Testing

Testing on hera.gnu against baseline ``develop-20211230`` shows that all tests pass except the following tests which fail due to missing baselines:

```
control_p8
control_debug_p8
cpld_control_c96_p8
cpld_debug_p8
```

Testing on hera.intel against baseline ``develop-20211230`` shows that all P8 tests fail due to missing baselines. All other tests (including ``cpld_control_p7_rrtmgp`` and ``cpld_bmark_p7``) pass. 

The forecast and restart files from the standalone P8 tests were spot-checked against the same file in the baseline P7 tests. The files were found to be identical.This is expected since the SDF is identical and CICE albedos are not used in the standalone.

The following oRTs were successful on hera.intel as the new P8 feature test suite. Logs were committed at d0297ad

- ``control_p8`` : bit_base, dbg_base, dcp, fhz, mpi, rst, thr
- ``cpld_control_p8`` : std_base, dcp, thr
- ``cpld_control_c98_p8`` : std_base, dcp, thr, rst, dbg_base
- ``cpld_bmark_p8`` : std_base, dcp, thr

After updating default variables for P8 at commit 07cd3eaf, the remaining p7 tests were again verified to be unchanged.

The remaining P7 tests (``control_p7_rrtmgp``, ``cpld_bmark_p7``,``cpld_control_p7_rrtmgp``) are updated for the final P7 configuration. The changed settings were

```
dddmp = 0.2  ->   0.1
cdmbgwd  = 0.14,1.8,1.0,1.0  -> 1.0,2.2,1.0,1.0
```

All P8 tests are updated to use

```
FV3_GFS_v16_coupled_p8 or FV3_GFS_v16_p8
dddmp = 0.1
cdmbgwd      = 1.0,2.2,1.0,1.0
iopt_sfc     = 3
min_seaice   = 1.0e-6 (coupled only) 
use_cice_alb = .true. (coupled only)
```

```
diff cpld_bmark_p7.input.nml cpld_bmark_p8.input.nml
161c161
<   iopt_sfc     = 1
---
>   iopt_sfc     = 3
182,183c182,183
<   min_seaice   = 1.0e-11
<   use_cice_alb = .false.
---
>   min_seaice   = 1.0e-6
>   use_cice_alb = .true.`
```

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [x] opnReqTest for newly added/changed feature (d0297ad)
- [x] CI (fd69d23)

## Dependencies

Required SDFs are dependent on 

- [x] https://github.com/ufs-community/ufs-weather-model/pull/984
- [x] https://github.com/NOAA-EMC/fv3atm/pull/465

### CODE REVIEW

**Comment by JessicaMeixner-NOAA** [comment]
Comparing the input.nml in the cpld_bmark_p8 test with what was run in p8a, I noted the following differences: 
```
<  cpld_bmark_p8
>  workflow p8a

<   dddmp = 0.2
>   dddmp = 0.1

<   cdmbgwd      = 1.1,0.72,1.0,1.0
>   cdmbgwd      = 1.0,2.2,1.0,1.0

<   iopt_sfc     = 1
>   iopt_sfc     = 3

<   min_seaice   = 1.0e-11
>   min_seaice   = 1e-6
```


The cdmbgwd will be updated in: https://github.com/ufs-community/ufs-weather-model/pull/969

PR #969 will also update the ICs to P7.2 not the P8a ICs, I don't know if we want to update them to p8a ICs here or not?

**Comment by DeniseWorthen** [comment]
@JessicaMeixner-NOAA I will update the ``min_seaice`` in default vars.

For ddmp, is the setting of 0.1 valid for all resolutions? 

For cdmbgwd, if it is to be changed, I'll need the values for other resolutions if they're also to be updated.

Regarding ``iopt_sfc``, the following is from an email last May. When did the desired value for ``iopt_sfc`` change? We actually do not have that defined as a variable. We have ``iopt_sfc = 1`` always. The value of  ``iopt_stc`` is a variable, currently set as 3.

**Comment by JessicaMeixner-NOAA** [comment]
@JongilHan66 or @yangfanglin can you answer @DeniseWorthen question about the ddmp setting?    The ddmp setting is the value we get from the workflow.   I'll dig into the workflow in the meantime to see if this parameter changes by resolution or not. 

The cdmbgwd value only needs to be changed for C384, see more details on PR: https://github.com/ufs-community/ufs-weather-model/pull/969).  I'm working through some issues I've encountered with reproducibility when changing this.  The errors I get don't make sense so perhaps I just did something wrong. 

The iopt_sfc was update to 3  in P8A  to correspond to NOAH-MP updates at the direction of @HelinWei-NOAA @barlage Info can be found on the P8 physics spreadsheet or here is the global-workflow issue: https://github.com/NOAA-EMC/global-workflow/issues/525

**Comment by JongilHan66** [comment]
I have no idea on the parameter ddmp. Fanglin may know this.

**Review by MinsukJi-NOAA — APPROVED** [review]

**Comment by DeniseWorthen** [comment]
Once we resolve the dddmp and cdmbgwd settings this PR will be ready for baseline creation. I did ask Moorthi and he said the dddmp can depend on resolution, so I think we need to resolve whether the non-c384 cases should remain 0.2 or use 0.1 instead.

**Comment by yangfanglin** [comment]
I'd suggest we use dddmp=0.1 for all resolutions.  
Set "cdmbgwd      = 1.0,2.2,1.0,1.0" for all resolutions for now.  The default cdmbgwd = 1.0,1.0,1.0,1.0 works for all resolutions as well.   This will likely change after we finalize the gravity-wave physics configuration for P8B.

**Comment by DeniseWorthen** [comment]
@yangfanglin Currently we're using resolution specific settings for CDMBGWD in all RTs (prototype or not). 

```
# resolution dependent settings
export CDMBWD_c48='0.071,2.1,1.0,1.0'
export CDMBWD_c96='0.14,1.8,1.0,1.0'
export CDMBWD_c192='0.23,1.5,1.0,1.0'
export CDMBWD_c384='1.1,0.72,1.0,1.0'
export CDMBWD_c768='4.0,0.15,1.0,1.0'
```

These should all be changed to cdmbgwd=1.0,2.2,1.0,1.0 ?

**Comment by JessicaMeixner-NOAA** [comment]
Just as an FYI some of the CDMBWD settings are hard-coded in input.nml files.

**Comment by yangfanglin** [comment]
@DeniseWorthen  Those CDMBGWD parameters in all current RTs works for the GFS.v16 based gravity-wave physics parameterization, the so-called uGWD.v0.   P7 and P8A have been using a newer version, uGWD.v1, for which only CDMBWD(2) is still controlled by the nemalist and should be set to 1.0 for all resolutions by design.  As I mentioned earlier we may make further changes to the GWD physics package in P8B.

**Comment by DeniseWorthen** [comment]
@yangfanglin I see. So the resolution dependent CDMBWD settings should remain in the RTs.  But for the P8 feature set, the default value for any resolution should be 1.0,2.2,1.0,1.0. 

The RTs for the P7 feature set used the resolution dependent settings I listed above. Any change in that parameter was never brought back to the RTs.

**Comment by yangfanglin** [comment]
"the resolution dependent CDMBWD settings" should only be used for P6 and earlier porotypes.    For P7 and  P8A, "1.0,2.2,1.0,1.0" should be applied for all resolutions.

**Comment by DeniseWorthen** [comment]
I understand. I'm more concerned now about what to do for the P7 RT tests we are maintaining. Those were never updated, even after the Prototype configuration was finalized. 

The point of maintaining the P7 bmark test after starting the new Prototype testing is to be able to compare against the final, previous prototype. If the P7 RTs are not actually using the final configuration, then we're not testing the right thing. 

I think I have no choice at this point but to update **both** the P7 tests we are carrying forward (cpld_bmark_p7, cpld_control_p7_rrtmgp, control_p7_rrtmgp) **and** create new P8 tests at the same time. 

@junwang-noaa I probably am not going to finalize all the tests by the end of the day.

**Comment by DeniseWorthen** [comment]
@JessicaMeixner-NOAA Can you please confirm that dddmp=0.1 was also used for the final P7 configuration?

**Comment by JessicaMeixner-NOAA** [comment]
Yes, dddmp=0.1 was used for P7.0 and P8A. It has been 0.1 for every prototype except dddmp=0.2 was used in P6.

File: tests/rt_35d.conf [hunk 1]
```diff
@@ -11,3 +11,12 @@ RUN     | cpld_bmark_p7_35d
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013040100
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013070100
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013100100
```
Comment (junwang-noaa) [inline tests/rt_35d.conf]: Do we want to turn off all the bmark_p7_35d tests?

**Comment by junwang-noaa** [comment]
@DeniseWorthen would you please also add Dusan's ecflow changes for issue #999 in tests/run_utils.sh:

     echo "ecflow_server is NOT running on ${ECF_HOST}:${ECF_PORT}"
     if [[ ${MACHINE_ID} == wcoss2 ]]; then
       # Annoying "Has NCO assigned port $ECF_PORT for use by this account? (yes/no) ".
 `-     echo yes | ${ECFLOW_START} -p ${ECF_PORT}`
 `+    echo yes | ${ECFLOW_START} -p ${ECF_PORT} -d ${RUNDIR_ROOT}/ecflow_server`
`     else`
 `-      ${ECFLOW_START} -p ${ECF_PORT}`
`+     ${ECFLOW_START} -p ${ECF_PORT} -d ${RUNDIR_ROOT}/ecflow_server`
`     fi`

File: tests/rt_35d.conf (hunk 1 above)
Comment (DeniseWorthen) [reply tests/rt_35d.conf]: Yes, I will remove the p7_35d tests and the edit the rt_35d to be only p8.

I will also remove the opnReq logs for p7.

**Review by SMoorthi-emc — APPROVED** [review]

File: tests/rt_35d.conf (hunk 1 above)
Comment (binli2337) [reply tests/rt_35d.conf]: @DeniseWorthen @junwang-noaa The cpld_bmark_p7_35d tests are used by the weekly tests.

**Review by DusanJovic-NOAA — APPROVED** [review]

File: tests/rt_35d.conf (hunk 1 above)
Comment (DeniseWorthen) [reply tests/rt_35d.conf]: @binli2337 I've updated the rt_weekly and set the fhmax to 5days. I set the wall clock to 120. I think that should be enough. Please check the changes, thanks

Comment (binli2337) [reply tests/rt_35d.conf]: @DeniseWorthen The updates are good. Thanks.

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: gnu
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/794674327/20220112144512/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_987
Baseline creation and move successful
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/794674327/20220112152637/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_12342
Test control 001 failed in run_test failed 
Test control_flake 005 failed in run_test failed 
Test control_rrtmgp 006 failed in run_test failed 
Test control_thompson 007 failed in run_test failed 
Test control_thompson_no_aero 008 failed in run_test failed 
Test control_ras 009 failed in run_test failed 
Test control_p8 010 failed in run_test failed 
Test rap_control 011 failed in run_test failed 
Test rap_2threads 012 failed in run_test failed 
Test rap_sfcdiff 014 failed in run_test failed 
Test hrrr_control 016 failed in run_test failed 
Test rrfs_v1beta 017 failed in run_test failed 
Test rrfs_conus13km_hrrr_warm 018 failed in run_test failed 
Test rrfs_conus13km_radar_tten_warm 019 failed in run_test failed 
Test control_debug 020 failed in run_test failed 
Test control_diag_debug 021 failed in run_test failed 
Test rap_control_debug 023 failed in run_test failed 
Test rap_diag_debug 024 failed in run_test failed 
Test rap_noah_sfcdiff_cires_ugwp_debug 025 failed in run_test failed 
Test rrfs_v1beta_debug 026 failed in run_test failed 
Test control_thompson_debug 027 failed in run_test failed 
Test control_thompson_no_aero_debug 028 failed in run_test failed 
Test control_thompson_extdiag_debug 029 failed in run_test failed 
Test control_rrtmgp_debug 030 failed in run_test failed 
Test control_ras_debug 031 failed in run_test failed 
Test control_stochy_debug 032 failed in run_test failed 
Test control_debug_p8 033 failed in run_test failed 
Test cpld_control_c96_p8 035 failed in run_test failed 
Please make changes and add the following label back:
hera-gnu-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: intel
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/794674327/20220112150013/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_13223
Test hafs_regional_docn 075 failed in run_test failed 
Test hafs_regional_docn_oisst 076 failed in run_test failed 
Test hafs_regional_datm_cdeps 077 failed in run_test failed 
Test compile_014 failed in run_compile failed 
Please make changes and add the following label back:
hera-intel-BL

**Comment by DeniseWorthen** [comment]
The Hera failures are disk quota issues.

**Review by junwang-noaa — APPROVED** [review]

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen can you please make this one line change for CI?
```
diff --git a/tests/ci/ci.test b/tests/ci/ci.test
index c39a5a9d..ac4bb63e 100644
--- a/tests/ci/ci.test
+++ b/tests/ci/ci.test
@@ -2,5 +2,5 @@ control
 thr mpi dcp rst bit dbg
 regional_control
 thr dcp
-cpld_control_c96_p7
+cpld_control_c96_p8
 rst dbg

```

**Comment by DeniseWorthen** [comment]
@JessicaMeixner-NOAA Would you please review? Thanks.

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Pull request merged**